src/settings/components/account-settings/confirm-delete-modal.tsx

```diff
@@ -0,0 +1,63 @@
+/*
+ * SPDX-FileCopyrightText: 2021 Zextras <https://www.zextras.com>
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+import React, { ReactElement } from 'react';
+import { TFunction } from 'i18next';
+import { Container, Text, CustomModal } from '@zextras/carbonio-design-system';
+import ModalHeader from '../folder-select-modal/modal-header';
+import ModalFooter from '../folder-select-modal/modal-footer';
+
+type DeleteConfirmProp = {
+	t: TFunction;
+	onConfirm: () => void;
+	setOpenModal: (value: boolean) => void;
+	openModal: boolean;
+};
+
```

Comment:
Instead of creating this component you can Also do the same with createModal function which is way easy for such modals

Reply:
i would also dare to say that it's preferrable